docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -66,66 +96,178 @@ To update the metadata and playback information on the server side:
 
 1. Create the media controller server handle using the `mc_server_create()` function:
 
-   ```
-   ret = mc_server_create(&g_server_h);
-   ```
+  ```
```

Comment:
please check the hyperlink in line 76

Reply:
@mijin85cho , thanks , I understood about symbolic link. That was a new learning for me. 
Thanks once again.